src/libraries/System.Private.CoreLib/src/System/CodeDom/Compiler/IndentedTextWriter.cs

```diff
@@ -254,5 +256,47 @@ public override void WriteLine(uint value)
             _writer.WriteLine(value);
             _tabsPending = true;
         }
+
+        public override async Task WriteLineAsync()
```

Comment:
Can you please add triple slash documentation for these new public APIs? Once these overrides are published, they will show up with empty documentation in MS Docs.

Right now, we [only see these methods](https://docs.microsoft.com/en-us/dotnet/api/system.codedom.compiler.indentedtextwriter?view=netcore-3.1) as inherited from TextWriter, so we only see the documentation that was added to TextWriter.

You can base your comments on the original TextWriter comments, and adjust the text for the specific behavior in these methods.

Reply:
@carlossanlop, we can't just auto-inherit the docs?

Reply:
Sure, we can auto-inherit, and then port the documentation using the DocsPortingTool (making sure to include inherited documentation from interfaces).
But shouldn't these methods have a slightly different documentation based on the fact that the text is indented?

Reply:
> But shouldn't these methods have a slightly different documentation based on the fact that the text is indented?

You're right.  It looks like the current overrides do.  Ok.